Gordon360/ApiControllers/HousingController.cs

```diff
@@ -25,22 +25,61 @@ public HousingController()
             _accountService = new AccountService(_unitOfWork);
         }
 
-        /// <summary>Check if the currently logged in user is authorized to view the housing staff page for applications</summary>
+        /// <summary>
+        /// Check if the currently logged in user is authorized to view the housing admin page
+        /// </summary>
         /// <returns></returns>
         [HttpGet]
-        [Route("staff")]
-        public IHttpActionResult CheckHousingStaff()
+        [Route("admin")]
```

Comment:
Good word choice. It makes sense to use 'admin' instead of 'staff' since not all of the staff members are authorized to view the page.